cerberus-web/src/main/java/com/nike/cerberus/util/CiphertextUtils.java

```diff
@@ -87,13 +78,8 @@ public static JsonObject toJsonObject(ParsedCiphertext parsedCiphertext) {
     for (KeyBlob keyBlob : parsedCiphertext.getEncryptedKeyBlobs()) {
       JsonObject blob = new JsonObject();
       blob.addProperty("providerId", keyBlob.getProviderId());
-      try {
-        blob.addProperty("providerInfo", new String(keyBlob.getProviderInformation(), "UTF-8"));
-      } catch (UnsupportedEncodingException e) {
-        LoggerFactory.getLogger("com.nike.cerberus.util.CiphertextUtils")
-            .error("Failed to create new string from key blob provider information", e);
-        throw new ApiException(DefaultApiError.INTERNAL_SERVER_ERROR);
-      }
+      blob.addProperty(
+          "providerInfo", new String(keyBlob.getProviderInformation(), StandardCharsets.UTF_8));
```

Comment:
if we pass StandardCharsets object instead of charset name as string to String constructor we could eliminate catching compiletime UnsupportedEncodingException